ext/bg/js/backend.js

```diff
@@ -800,6 +802,82 @@ class Backend {
 
     // Utilities
 
+    _getBrowserIconTitle() {
+        return (
+            chrome.browserAction !== null &&
+            typeof chrome.browserAction === 'object' &&
+            typeof chrome.browserAction.getTitle === 'function' ?
+            new Promise((resolve) => chrome.browserAction.getTitle({}, resolve)) :
+            Promise.resolve('')
+        );
+    }
+
+    _updateBadge() {
+        let title = this._defaultBrowserActionTitle;
+        if (
+            title === null ||
+            chrome.browserAction === null ||
+            typeof chrome.browserAction !== 'object'
```

Comment:
This could use `!isObject(chrome.browserAction)`. Also applies to the earlier comment about `_getBrowserIconTitle`

Reply:
0628c333d9b0fc15aeb2ec8cdead9a8dff4fb087